lib/services/android-debug-service.ts

```diff
@@ -186,7 +193,28 @@ class AndroidDebugService implements IDebugService {
 
 			this.device.applicationManager.stopApplication(packageName).wait();
 			this.device.applicationManager.startApplication(packageName).wait();
-
+			
+
+			let waitText: string = `0 /data/local/tmp/${packageName}-debugbreak`;
+			let maxWait = 6;
+			let debugerStarted: boolean = false;
+			while (maxWait > 0 && !debugerStarted) {
+				let forwardsResult = this.device.adb.executeShellCommand([`ls -s /data/local/tmp/${packageName}-debugbreak`]).wait();
+				maxWait--;
+				debugerStarted = forwardsResult.indexOf(waitText) === -1;
+				if (!debugerStarted) {
+					this.sleep(500);
+				}	
+			}
+			
+			if (debugerStarted) {
+				console.log("# NativeScript Debugger started #");
+			}
+			else {
+				console.log("# NativeScript Debugger did not start in time #");
```

Comment:
lets not over engineer a simple console output, exactly how is "NativeScritp Debugger started" message complicating user's workflow.

otherwise updated the code as per your suggestions